### ISSUE TITLE
Segmentation Fault while running compiled version of my Application Developed with Kivy on Linux (POP OS 22.04 precisely)

### ISSUE THREAD

**Issue by kengoon** [Nuitka/Nuitka#1739]
the segfault occurs whenever the app tries to access a `kiy.uix.widget.Widget` class from `kivy.factory.Factory` class
here is a little code example to explain better

# Case 1

```python
from kivy.app import App
from kivy.factory import Factory
from kivy.uix.button import Button
from kivy.lang import Builder
Builder.load_string('''
<CustomButton@Button>:
    text: "Hello World"
''') # sometimes it occurs here if the widget is declared as root widget

custom_button = Factory.CustomButton() # segfault occurs here and the app crashes too with no other error

class TestApp(App):
    def build(self):
        return custom_button

TestApp().run()
```

# Case 2

Lets assume that I don't want to use factory to access my custom widget. I will still get segfault

```python
from kivy.app import App
from kivy.factory import Factory
from kivy.lang import Builder
Builder.load_string('''
<CustomButton>:
    text: "Hello World"
''') # sometimes it occurs here if the widget is declared as root widget


class CustomButton(Button):
    pass


custom_button = CustomButton() # segfault occurs here and the app crashes too with no other error

class TestApp(App):
    def build(self):
        return custom_button

TestApp().run()
```

# EDIT
further test shows that trying to instantiate some kivy widget like `Button, Label ..` results to segfault. from the test I have made so far I think any widget that has to go through `kivy.factory.Factory` class results to segfault even without creating a custom widget

**Comment by kayhayen** [comment]
In the template, I have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that I do not want to be left guessing. For you it's easy to erase the issue template and expect me to ask you questions only as necessary, but I cannot do that. I would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.

In short: No issue template, no fun. Please provide the info, or close the issue, after which I would delete it.

**Comment by kengoon** [comment]
> In the template, I have made it clear, that not filling out the issue template is not acceptable. I am closing this kind of report as invalid. The main reason is that I do not want to be left guessing. For you it's easy to erase the issue template and expect me to ask you questions only as necessary, but I cannot do that. I would much rather wait for you to create a new report, fill out the template, or wait for somebody with the necessary skill to do it.
> 
> In short: No issue template, no fun. Please provide the info, or close the issue, after which I would delete it.

Thank you for at least responding to this issue, I will go through the issue template and put things in place. I apologise for making things harder for you.

**Comment by kayhayen** [comment]
Yeah, sorry only catching up now, and reports that are not actionable, I used to just ignore them, now I tend to just delete them. I will with this one now, feel free to re-open with details.
